ledger/participant-integration-api/src/main/scala/platform/localstore/PersistentPartyRecordStore.scala

```diff
@@ -52,9 +52,9 @@ class PersistentPartyRecordStore(
 
   private val logger = ContextualizedLogger.get(getClass)
 
-  override def createPartyRecord(partyRecord: domain.ParticipantParty.PartyRecord)(implicit
+  override def createPartyRecord(partyRecord: domain.PartyRecord)(implicit
```

Comment:
Suggest to use consistently either `domain.PartyRecord` or `PartyRecord` but not both in a single file